### ISSUE TITLE
docker: telepites

### ISSUE THREAD

**Issue by TheNON75** [szabbenjamin/digionline#52]
Szia/Sziasztok,

docker segitsegevel szeretnem hasznalni a programot (xteve/plex), de nem allitanam, hogy magas szintu ismeretekkel rendelkezem.
Meglevo image-et termeszetesen tudok hasznalni (docker-compose segitsegevel is), azonban ott meg nem tartok, hogy magam dobjak ossze egy buildet.
2 docker image-et talaltam: norberttj es progie alatt, de nem egyertelmu, hogy melyiket kellene hasznalni, ha egyaltalan ezek valamennyire hivatalosnak tekinthetok.

A wiki-ben az alabbiakra nem talaltam valaszt:
- docker mappa, amiben a szukseges m3u8 file es musorujsag talalhato
- host v bridge a javasolt halozati beallitas (eddig csak bridge szerepelt a peldakban)

Tehat 2 fo kerdesre szeretnek valaszt illetve segitseget kapni:
- melyik a javasolt docker image
- javasolt container beallitasok

Elore is köszönöm

**Comment by gudszent** [comment]
dockerfile végén megtalálod, de
Ha docker manage-hez használsz portainer, abban 2 katt az image létrehozás. Abban tudok segíteni, illetve:
vgy
# Build docker image: 
#   sudo docker build -t digionline https://github.com/szabbenjamin/digionline.git

    docker mappa, amiben a szukseges m3u8 file es musorujsag talalhato
-nem kell, http-n leszedi a plex is szerinte, nekem Emby van
    host v bridge a javasolt halozati beallitas (eddig csak bridge szerepelt a peldakban)
-bridge elég

Tehat 2 fo kerdesre szeretnek valaszt illetve segitseget kapni:

    melyik a javasolt docker image
lásd első részt:)
    javasolt container beallitasok
# Create and run docker container: 
#   sudo docker run -d -p 9999:9999 --restart unless-stopped --env DOMAIN=valami.local --env EMAIL=a@b.hu --env PASSWORD=jelszo --name container-digionline digionline
Plexhez is ez a 2 kell majd
# Kodi PVR IPTV Simple Client addon 
#   TV channel list: http://localhost:9999/channels_IPTV.m3u8
#   TV EPG source: http://localhost:9999/epg.xml

**Comment by gudszent** [comment]
> 
> 
> Szia @gudszent,
> 
> nagyon szépen köszönöm a válaszokat.
> A build nekem sajnos nem müködik a peldaban leirtakkal ellentétben (NAS-on futtatom a dockert es a git kiterjesztést egyelőre nem szereti)
> 
> Bár időközben Gazsó készitett egy friss docker image-et, köszönettel venném a felkinált segitséget Portainerhez, így pár dologgal tudnék kisérletezni, pl az imagehez adni az ffmpeg-et a tvheadend url támoagtásához.
> 
> ui.: a plex sajnos már nem kezel plugineket, a helyzet így bonyolultabb lett kicsit
> 
> Előre is köszönöm

Próbáld ki plex helyett Emby-t és akkor nem kell szenvedned az img-el ;)

**Comment by TheNON75** [comment]
Remek, köszi, de az adott mindenható  bocsássa meg a bunomet, hogy nas-t hasznalok ;)

**Comment by gudszent** [comment]
> 
> 
> Remek, köszi, de az adott mindenható bocsássa meg a bunomet, hogy nas-t hasznalok ;)

Semmi gond, Én is, bár ez nem tom hogy jön ide. :D
Az emby és a plex kb ua.

Magát az image-et lebuildelni, ami a repoban megvan, annak a linkjét csak rakd be ide:
![kép](https://user-images.githubusercontent.com/6563633/110023392-72536380-7d2d-11eb-8992-69059014e817.png)
Többit intézi a portainer

**Comment by TheNON75** [comment]
A nas-os megjegyzés csak annyiból volt lényeges, hogy ha nem tévedek, muszáj dockerben futtatnom
Köszönöm, pár óra és megnézem

**Comment by gudszent** [comment]
> 
> 
> A nas-os megjegyzés csak annyiból volt lényeges, hogy ha nem tévedek, muszáj dockerben futtatnom (szethackelni nem akarom), így ha emby ha nem, szukseg van img-re
> 
> Köszönöm, pár óra és megnézem

Nem tudom milyen NAS-od van, de Emby-t is lehet dockerben futtani de natívan telepíteni is...

**Comment by TheNON75** [comment]
Úgy értettem, hogy a digionline-t futtatom dockerben. Ebédszünetet van, mindjárt megnézem az emby-t. Legfeljebb az lesz a tv nezos app (syno  ds412+)

**Comment by droM4X** [comment]
az embynel akarcsak a plexnel vannak fizetos reszek
amikor behoztak a paywallt az emby-be, akkor forkoltak es csinaltak egy forever-free alternativat neki jellyfin neven
kb. ugyanazt tudja, meg hasonloan is nez ki
egy ertelmes igyenes alternativa, ha mar erre van szukseg

**Comment by TheNON75** [comment]
köszi @droM4X, app most fejeltem le a paywall-t :) Pley-re par eve befizettem, egyelore "joazúgy..."
Jellyfin-t is probaltam par hete, de nem nagyon valt be, hol elindult az adas hol nem. Tobbnyire hol nem :)
Nem adom fel, megnezem jellyt ujra

**Comment by droM4X** [comment]
kodi nem jatszik semmilyen modon? azzal lenne a legegyszerubb megoldani, es a plex is kivalthato vele (jellyfin-re), vagy gondolom a plex is kapcsolhato valamilyen modon a kodihoz.

kodira csak mint medialejatszo gondolj, nem muszaj annak lennie a mediaszervernek (nekem is ezert van kulon jellyfin, mert tobb helyrol is lehet ra kapcsolodni).
simpleiptv pvr plugint kell betolni a kodiba, az eredeti leiras is ehhez keszult.
teljesen jol mukodik (most milyen arrol nem tudok nyilatkozni, sajat forkot hasznalok, az teljesen jo, nyilvan a digionline stream hibaival egyutt: kockasodik a kep, neha nincs hang, stb.)

**Comment by TheNON75** [comment]
mindenem headless fut, de meg fogom probalni a kodi/osmc verziot is. (a nas foleg)
visszajelzek

**Comment by gudszent** [comment]
Srry, Az emby-re sok éve befizettem, így nem tudom merre jár a paywall, erről mindig megfeledkezek.
Jellyfin-t anno forknál Én is kipróbáltam, de a kliensek hiánya(pl lg webos re akkor nem volt), illetve már úgyis kifizetve alapon maradt az emby :)
Nekem is minden konténerben fut, szóval az nem lehet gond, de vannak adók amik nehezen indulnak, pl amc

**Comment by TheNON75** [comment]
igen, tenyleg nehany nehezebben indul. Azt kell felteteleznem, hogy minden rendben mukodik, kiveve a digi szolgaltatasa.
Sajat appal is elhasalnak a streamek neha es whatsapp hivasnal a digis fél kapcsolata elegge ramaty neha. Valoszinuleg a kulfoldi savszelesseg eleg karcsu digieknel.
Mindenesetre orulok annak, ami van, neha nem mukodik, de amikor igen, akkor boldogsag, így ha meglatogatnak a nagyik, tudnak hiradot stb nezni, a lanyunk meg, ha eljon az ideje, meset es termeszetfilmet.

Köszönöm mindenkinek
